### ISSUE TITLE
TeX names

### ISSUE THREAD

**Issue by fionn** [fionn/feynman#4]
The `Particle` class has an unused `tex` property for typesetting particle names. Let's fill this out for concrete classes. It would be especially nice if we can do this implicitly in some cases, e.g. antiparticles and neutrinos, where the symbol changes in predictable ways.

**Comment by neal-o-r** [comment]
Question on this, do we want to have ```name```, ```symbol```, and ```tex``` instances, especially in the case that we'll be setting symbol equals name in a lot of cases? I've populated the tex field for the ones we've got anyway, but we could decide going forward to write the symbol in tex notation, rather than unicode or whatever.

**Comment by fionn** [comment]
Yeah, let's just ditch the `symbol` property. We don't need to be so generic.
